### PR TITLE
FP-679: Catch-up Prototype to current Facade functionality

### PR DESCRIPTION
**Pull request by m-asim-iqbal**
- Dataset edit page added with static data.
- On pressing the Delete button, a box appears for confirmation.
- The edit button takes the page to a static edit dataset page.
- Login Page added.
- Login Error Page added.
- User Dashboard Page added
- Publisher Requested Page added

Please review this PR:
@dynamicmonkeys @radicalgeek

### CODE REVIEW

File: app/config.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "serviceName": "Find Transport Data"
+  "serviceName": "Find Transport Data - GOV.UK"
```
Comment (dynamicmonkeys) [inline app/config.json:2]: This should be stuck as `Find Transport Data` to reflect the GDS Service Name

**Review by dynamicmonkeys — CHANGES_REQUESTED** [review]
Just one change & a Q on Slack re: select component

File: app/config.json (hunk 1 above)
Comment (m-asim-iqbal) [reply app/config.json:2]: On the current prod (CKAN) the page title is "Find Trasport Data - GOV.UK"

Comment (dynamicmonkeys) [reply app/config.json:2]: Hi, I've compared a number of services and the guidance and there's no definitive answer anymore!

**Pull request merged**